src/Chart.PieceLabel.js

```diff
@@ -184,6 +184,7 @@
       this.showZero = pieceLabel.showZero;
       this.overlap = pieceLabel.overlap;
       this.images = pieceLabel.images || [];
+      this.padding = pieceLabel.padding || 2;
```

Comment:
1. since it's position outside only, maybe call outsidePadding?
2. I think we can let default is fontSize * 1.5 + 2, and users can set any value

Reply:
Thanks.
Do I have to do anything about this pull request?

Reply:
or I can change this when I have time.